src/vellum/workflows/nodes/displayable/tool_calling_node/utils.py

```diff
@@ -250,10 +250,6 @@ def _hydrate_composio_tool_definition(tool_def: ComposioToolDefinition) -> Compo
             toolkit=toolkit_slug.upper() if toolkit_slug else tool_def.toolkit,
```

Comment:
just update the return type of this method to return the `FunctionDefinition` directly, so that we could keep `parameters=tool_details.get("input_parameters",{})`